lib/logflare_web/live/alerts/components/alert_form.heex

```diff
@@ -0,0 +1,79 @@
+<div class="tw-flex tw-w-full tw-my-10 tw-gap-2">
```

Comment:
wrong file name, should be `lib/logflare_web/live/alerts/components/alert_form.html.heex`